packages/babel/package.json

```diff
@@ -13,10 +13,14 @@
   "author": "Rich Harris",
   "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel#readme",
   "bugs": "https://github.com/rollup/plugins/issues",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/es/index.js"
```

Comment:
I wonder - is there any reason why `require` is put first here? I usually put the `import` as one of the first conditions here. Is it purely an microoptimization that assumes that most people will be loading this from CJS?

Reply:
No, it is just coincidence. I do not think it makes any difference as the conditions are mutually exclusive.

Reply:
Right, they are - although for a CJS package I usually go with this: `exports: { import, default }`. We declare support for node14+ so we probably don't need to worry about `require` not being supported in some node12 versions. So I guess this is fine.

Reply:
Ah right, then your version IS superior even though it pains me that the default would be CJS... I mean, at least it does not hurt to write it this way. By the way I am actively considering if with Rollup 4, we should not follow Sindre's lead and convert the entire Rollup ecosystem to ESM only. But that is me thinking aloud for now and definitely something to be discussed elsewhere.